SignUp/SignUp/SIgnUpView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SIgnUpView.swift
+//  SignUp
+//
+//  Created by Булат Хабибуллин on 17.09.2020.
+//  Copyright © 2020 Bulat Khabibullin. All rights reserved.
+//
+
+import UIKit
+
+protocol SignUpViewDelegate: AnyObject {
+    func registration(login: String, password: String)
+    func switchSignUpView()
+}
+
+class SignUpView: UIView {
+    @IBOutlet var loginTextField: UITextField!
+    @IBOutlet var passTextField: UITextField!
+    @IBOutlet var confPassTextField: UITextField!
+    
+    weak var delegateUp: SignUpViewDelegate?
```

Comment:
Правильнее просто называть delegate.

```suggestion
    weak var delegate: SignUpViewDelegate?
```